includes/Data/Brands.php

```diff
@@ -6,61 +6,110 @@
-						'utm_source' 	=> 'onboarding',
-						'utm_medium' 	=> 'wp_admin',
-						'utm_campaign' 	=> 'wp-setup')),
-		),
-		'bluehost-india' => array(
-			'brand'            => 'bluehost-india',
-			'name'             => 'Bluehost',
-			'url'              => 'https://bluehost.in',
-			'knowledgeBaseUrl' => 'https://www.bluehost.in/hosting/help/results/wordpress',
-			'helpUrl'          => 'https://www.bluehost.in/hosting/help',
-			'blogUrl'          => 'https://www.bluehost.in/tutorials',
-			'facebookUrl'      => 'https://www.facebook.com/BlueHostIndia',
-			'twitterName'      => '@BluehostIndia',
-			'twitterUrl'       => 'https://twitter.com/bluehostindia',
-			'youtubeUrl'       => null,
-			'linkedinUrl'      => null,
-			'accountUrl'       => 'https://my.bluehost.in',
-			'domainsUrl'       => 'https://my.bluehost.in/hosting/app?lil=1#/domains',
-			'emailUrl'         => 'https://my.bluehost.in/hosting/app?lil=1#/email-office',
-			'hireExpertsInfo'   => array(
-				'defaultLink' 		=> 'https://www.bluehost.in/solutions/full-service', 
-				'utmParameters' 	=> array(
+	protected static function brands_info() { 
+		return array(
+			'bluehost'       => array(
+				'brand'            => 'bluehost',
+				'name'             => 'Bluehost',
+				'url'              => 'https://bluehost.com',
+				'knowledgeBaseUrl' => 'https://www.bluehost.com/help/results/wordpress',
+				'helpUrl'          => 'https://www.bluehost.com/help',
+				'blogUrl'          => 'https://www.bluehost.com/blog/',
+				'facebookUrl'      => 'https://www.facebook.com/bluehost',
+				'twitterName'      => '@bluehost',
+				'twitterUrl'       => 'https://twitter.com/bluehost',
+				'youtubeUrl'       => 'https://www.youtube.com/user/bluehost',
+				'linkedinUrl'      => 'https://www.linkedin.com/company/bluehost-com',
+				'accountUrl'       => 'https://my.bluehost.com',
+				'domainsUrl'       => 'https://my.bluehost.com/hosting/app?lil=1#/domains',
+				'emailUrl'         => 'https://my.bluehost.com/hosting/app?lil=1#/email-office',
+				'hireExpertsInfo'   => array(
+					'defaultLink'		=> 'admin.php?',
+					'fragment'			=> '#/marketplace/services/blue-sky', 
+					'queryParameters' 	=> array(
+						'page'			=> 'bluehost',
+						'utm_source'	=> 'onboarding',
+						'utm_medium'	=> 'wp_admin',
+						'utm_campaign'	=> 'wp-setup'
+					),
+				),
+				'expertsInfo' => array(
+					'defaultLink'		=> 'https://my.bluehost.com/cgi/app/#/marketplace/product/i/bluesky',
+					'queryParams'		=> array(
+						'utm_source'	=> 'wp-onboarding',
+						'utm_medium'	=> reset(explode('/', str_replace(WP_PLUGIN_DIR . '/', '', __DIR__))),
+					),
+				),
+				'fullServiceCreativeTeamInfo' => array(
+					'defaultLink'		=> 'https://www.bluehost.com/solutions/full-service',
+					'fragment'			=> '#full-service',
+					'queryParams'		=> array(
+						'utm_source'	=> 'wp-onboarding',
+						'utm_medium'	=> reset(explode('/', str_replace(WP_PLUGIN_DIR . '/', '', __DIR__))),
```

Comment:
![Screenshot 2022-12-12 at 10 23 43 AM](https://user-images.githubusercontent.com/38878906/206964606-08f55466-6b18-4839-b4dd-c72e0d1456b4.png)
Empty `utm_medium`, again referring to my previous comment let's use the value from `container` here. I see this logic is used in many places so not gonna comment again let's change any such places to use `container`

Reply:
fixed